hamsm_restarting/untar.sh

```diff
@@ -0,0 +1,30 @@
+# This *intentionally* is not marked executable, and does not have a hashbang..
+# There's something weird that happens with backgrounding that tar process, at
+#       least on the machine I'm developing this on, and launching with `./untar.sh`
+#       instead of `bash untar.sh`
+
+
+echo "This should be run from the haMSM tutorial WEST_SIM_ROOT!"
+
+echo "Untarring $1"
+
+tar --totals=USR1 -xzf $1 &
```

Comment:
```suggestion
tar --totals -xzf $1 &
```
The `--totals=USR1` is causing a lot of grief on my M1 macbook pro. Stating: `tar: Option --totals does not allow an argument`.  Running `bsdtar 3.5.1 - libarchive 3.5.1 zlib/1.2.11 liblzma/5.0.5 bz2lib/1.0.8 `

Reply:
I'll just remove the progress tracking